### PR TITLE
Multi-value variable info added

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request (PR) ...

Issue number: [DASH-383](https://sumologic.atlassian.net/browse/DASH-383)

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/dashboards-new/filter-template-variables.md [hunk 1]
```diff
@@ -80,11 +84,14 @@ To add a Metrics Metadata Search variable to a dashboard, do the following:
...
-1. Optional: Toggle ON **Include the option to select all values (\*)** to include a wildcard asterisk (\*) in the available options.
-1. Optional: Provide a **Default Value** for the variable.
+1. The **Fields (key)** is the metadata field you want to use as the filter.
+1. (Optional) **Filters**. Allows you to filter the scope of your data before choosing the field you want to use. Click in the **Filter** input, begin typing, and choose a filter from the pop-up list. A list of valid values appears. Select a value for the filter, and add other filters as needed.
+1. (Optional) **Include the option to select all values**. Toggle ON to include a wildcard asterisk (\*) in the available options.
+1. (Optional) **Allow multiple values**. Variables will be single value by default, by toggling ON you can add more than one values. By default, * will be used if no values are added after toggling ON.
+:::note
+You can add a maximum of 20 values.
+:::
```
Comment (kimsauce) [inline docs/dashboards-new/filter-template-variables.md:93]: need to indent this block and line 122

Comment (JV0812) [reply docs/dashboards-new/filter-template-variables.md]: Thanks Kim, I have updated accordingly.

**Review by kimsauce — APPROVED** [review]

**Comment by jpipkin1** [comment]
@JV0812, is this PR ready to publish?

**Comment by JV0812** [comment]
No John, the UI is not ready still.

File: docs/dashboards-new/filter-template-variables.md [hunk 2]
```diff
@@ -51,12 +51,16 @@ To add a Custom List variable to a dashboard, do the following:
...
 
-![Create Template Variable dialog.png](/img/dashboards-new/filter-template-variables/Create-Template-Variable-dialog.png)
+    ![Create Template Variable dialog.png](/img/dashboards-new/filter-template-variables/Create-Template-Variable-dialog.png)
 
 1. Select **Custom List** as the **Variable Type**.
 1. **List Items** are your variable values. Use a comma separated list for variable options, separating individual options with a comma. For example, `small, medium, large`.
-1. Optional: Provide a **Default Value** for the variable.
-1. Optional: Toggle ON **Include the option to select all values (\*) **to include a wildcard asterisk (\*) in the available options.
+1. (Optional) **Default Value**. Provide a **Default Value** for the variable.
+1. (Optional) **Include the option to select all values**. Toggle ON to include a wildcard asterisk (\*) in the available options.
```
Comment (oshoghi) [inline docs/dashboards-new/filter-template-variables.md:59]: Can you remove this line. We'd decided to remove this toggle altogether. You may also have to update some of your screenshots.

File: docs/dashboards/filter-template-variables.md [hunk 3]
```diff
@@ -203,3 +204,8 @@ To use a metrics variable in a query working in advanced query mode, do the fol
 
 1. Locate the name of the variable you'd like to use in the dropdown list and copy it.
 1. Go to your metrics query and insert *{{ var_name }}* in the query, substituting `var_name` with the actual name of the variable.
+
+
+## Limitation
+
+- For log search with multi-value variables, do not add multi-value variables `{{variable}}` before the first pipe, instead you can add the variables after the first pipe to refine the search results or you can add the value by itself before the first pipe.
```
Comment (jpipkin1) [inline docs/dashboards/filter-template-variables.md:211]:
```suggestion
For log search with multi-value variables, do not add multi-value variables `{{variable}}` before the first pipe, instead you can add the variables after the first pipe to refine the search results or you can add the value by itself before the first pipe.
```

**Review by jpipkin1 — APPROVED** [review]
Looks good, @JV0812! I made one tiny change.

File: docs/dashboards/filter-template-variables.md [hunk 4]
```diff
@@ -203,3 +204,8 @@ To use a metrics variable in a query working in advanced query mode, do the fol
 
 1. Locate the name of the variable you'd like to use in the dropdown list and copy it.
 1. Go to your metrics query and insert *{{ var_name }}* in the query, substituting `var_name` with the actual name of the variable.
+
+
+## Limitation
```
Comment (oshoghi) [inline docs/dashboards/filter-template-variables.md:209]: @JV0812 Couple of comments here. There's already a limitations section at the top of the page. Second the wording is quite right. You can use a multi-variable in the "selector" but only in the form of `<field>={{variable}}`. Adding `<field> matches "{{variable}}"` will only work after the first pipe.

Comment (JV0812) [reply docs/dashboards/filter-template-variables.md]: @oshoghi Updated, can you please check now.

File: docs/dashboards/filter-template-variables.md [hunk 5]
```diff
@@ -16,10 +17,11 @@ This feature requires two parts to work:
...
 
-* It is updated every night for the last 24 hours starting from 10PM PT to the previous day at 10PM PT. 
-* Only dashboards that were viewed in the last 3 days are updated.
-* Up to 10,000 log values and 1,000 metric values are displayed.
-* Values for template variables are based on the time range of the Dashboard.
+- It is updated every night for the last 24 hours starting from 10PM PT to the previous day at 10PM PT. 
+- Only dashboards that were viewed in the last 3 days are updated.
+- Up to 10,000 log values and 1,000 metric values are displayed.
+- Values for template variables are based on the time range of the Dashboard.
+- For log search with multi-value variables, you can use a multi-variable in the **selector** but only in the form of `<field>={{variable}}`. Adding `field` matches `{{variable}}` will only work after the first pipe.
```
Comment (oshoghi) [inline docs/dashboards/filter-template-variables.md:24]: @JV0812 please change the second example to one code block: "Adding `<field> matches {{variable}}`"

**Comment by JV0812** [comment]
Closing this PR as this feature is still under development and will take some more time. Will create a new PR when the work for this feature starts back.

**Pull request closed without merging**